clients/privacy-center/cypress/e2e/consent-i18n.cy.ts

```diff
@@ -902,6 +908,256 @@ describe("Consent i18n", () => {
+            visitDemoWithI18n({
+              navigatorLanguage: ENGLISH_LOCALE,
+              globalPrivacyControl: true,
+              fixture: "experience_banner_modal.json",
+            });
+          });
+          it("applies experience language overrides", () => {
+            const experienceTranslationOverrides = {
+              fides_title: "My override title",
+              fides_description: "My override description",
+              fides_privacy_policy_url: "https://example.com/privacy",
+              fides_override_language: "en",
+            };
+            cy.fixture("consent/experience_banner_modal.json").then(
+              (experience) => {
+                const experienceItem = experience.items[0];
+                const translation: ExperienceConfigTranslation =
+                  experienceItem.experience_config.translations.filter(
+                    (i: ExperienceConfigTranslation) => i.language === "en"
+                  )[0];
+                stubConfig(
+                  {
+                    options: {
+                      customOptionsPath: TEST_OVERRIDE_WINDOW_PATH,
+                    },
+                    experience: experienceItem,
+                  },
+                  null,
+                  null,
+                  undefined,
+                  { ...experienceTranslationOverrides }
+                );
+                cy.get("div#fides-banner").within(() => {
+                  cy.get("div.fides-banner-title").contains(
+                    translation.banner_title as string
+                  );
+                  cy.get(
+                    "div#fides-banner-description.fides-banner-description"
+                  ).contains(translation.banner_description as string);
+                  cy.get("#fides-privacy-policy-link a").should(
+                    "have.attr",
+                    "href",
+                    experienceTranslationOverrides.fides_privacy_policy_url
+                  );
+                });
+                // Open the modal
+                cy.contains("button", "Manage preferences").click();
+                cy.get("div#fides-modal").within(() => {
+                  cy.get(".fides-modal-title").contains(
+                    experienceTranslationOverrides.fides_title
+                  );
+                  cy.get(".fides-modal-description").contains(
+                    experienceTranslationOverrides.fides_description
+                  );
+                });
+              }
+            );
+          });
+        });
+        describe("when fides_override_language is only part of an experience locale string", () => {
+          beforeEach(() => {
+            visitDemoWithI18n({
+              navigatorLanguage: FRENCH_LOCALE,
+              globalPrivacyControl: true,
+              fixture: "experience_banner_modal.json",
+            });
+          });
+          // TODO (PROD-1885): matchLocale needs to support partial language match
```

Comment:
ticket here- https://ethyca.atlassian.net/browse/PROD-1885. @NevilleS or @mfbrown can you tell me how urgent support for this is?

Reply:
Left some notes on that ticket, but in general I think we should simplify it and not do partial matches at all when overriding translations 👍

Reply:
You can nuke this test completely I think